.github/workflows/publish.yml

```diff
@@ -43,3 +44,26 @@ jobs:
       - run: npm whoami; npm --ignore-scripts publish --provenance --access public
         env:
           NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+  publish-github:
+    name: Publish to GitHub Packages
+    runs-on: ubuntu-latest
+    needs: [bump-version, create-release]
+    permissions:
+      packages: write
```

Comment:
I think you might need `contents: read` as well to run `npm ci` 😊

Reply:
Ah thanks for spotting that! I just updated 🙌🏼